2019-03-01 13:46:03.973916: [ligne 86 / code 410 / erreur <class 'KeyError'>] - 'point_technique_path'
Traceback (most recent call last):
  File "C:\Applications\Controleur\python\controleur_param.py", line 86, in lancer_controles
    verif_point_technique_c3a(list_controle_exe[5])
  File "C:\Applications\Controleur\python\controles.py", line 559, in verif_point_technique_c3a
    shape,list_point_technique = get_shape(conf["point_technique_path"],shape_point_technique_nom)
KeyError: 'point_technique_path'


